### PR TITLE
映射文档 add cov etc.

### PR DESCRIPTION
**Pull request by zpceng314**
add cov etc.

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.adjoint.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+### 转写示例
+
+```python
+# Pytorch 写法
+y = input.adjoint()
+
+# Paddle 写法
+perm = list(range(input.ndim))
+perm[-1], perm[-2] = perm[-2], perm[-1]
+y = paddle.conj(paddle.transpose(input, perm=perm))
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.adjoint.md]: 写API映射关系：人工转换思路；可以比自动转换思路灵活点，重在思路提示。
参考torch.transpose

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.adjoint.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+## [ 组合替代实现 ]torch.adjoint
+
+### [torch.adjoint](https://pytorch.org/docs/stable/generated/torch.adjoint.html#torch.adjoint)
+```python
+torch.adjoint(input)
+```
+
+Paddle 无此 API，需要组合实现。
+
+### 转写示例
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.adjoint.md:10]: 同 torch.adjoint

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.digamma.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+|  input  |  x  | 表示输入的 Tensor ，仅参数名不一致。  |
+|  out  | -  | 表示输出的 Tensor ， Paddle 无此参数，需要进行转写。    |
+
+### 转写示例
+#### out：指定输出
+```python
+# Pytorch 写法
+torch.special.digamma(input), out=y)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.digamma.md]: 这个多了个 `)`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.hardshrink.md [hunk 4]
```diff
@@ -6,10 +6,10 @@
...
 
-### [paddle.Tensor.hardshrink]()
+### [paddle.nn.functional.hardshrink](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/hardshrink_cn.html#hardshrink)
 
 ```python
-paddle.Tensor.hardshrink(threshold=0.5, name=None)
+paddle.nn.functional.hardshrink(x, threshold=0.5, name=None)
 ```
 
 两者功能一致且参数用法一致。
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.hardshrink.md]: 仅参数名不一致

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.norm.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+```python
+paddle.linalg.norm(x, p='fro', axis=None, keepdim=False, name=None)
+```
+
+Pytorch 与 Paddle 参数不一致，具体如下：
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| input | x         | 表示输入的一个 tensor 列表 ，仅参数名不一致。                    |
+| ord | p         | 范数的种类。参数不一致。Pytorch 支持负实数的范数，Paddle 暂不支持，暂无转写方式。                   |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.norm.md:20]: 这个还是写 `torch参数更多` 吧，功能diff缺失之处，严格上不作为参数不一致

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.cov.md [hunk 6]
```diff
@@ -19,12 +19,12 @@ paddle.linalg.cov(x,
...
 
-两者功能一致且参数用法一致，仅参数名不同，具体如下：
-### 参数差异
+仅 paddle 参数更多，具体如下：
+### 参数映射
 | PyTorch       | PaddlePaddle | 备注                                                   |
 | ------------- | ------------ | ------------------------------------------------------ |
 | <font color='red'> input </font>         | <font color='red'> x </font>            | 输入的 Tensor 。                   |
-| <font color='red'> correction </font>    | <font color='red'> ddof </font>          | 若为 True ，返回无偏估计结果；若为 False ，返回普通平均值计算结果。 |
+| <font color='red'> correction </font>    | <font color='red'> ddof </font>          | 若为 True ，返回无偏估计结果；若为 False ，返回普通平均值计算结果。 Paddle 暂不支持 correction 不为`0`或`1`的情况。|
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.cov.md]: 这个看不太清晰，是说 torch支持非bool的情况吗

Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.cov.md]: torch支持更多的整数，但是paddle只支持bool

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.adjoint.md (hunk 1 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.adjoint.md]: done!

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.adjoint.md (hunk 2 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.adjoint.md:10]: done!

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.digamma.md (hunk 3 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.digamma.md]: done!

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.hardshrink.md (hunk 4 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.hardshrink.md]: done！

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.norm.md (hunk 5 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.norm.md:20]: done！

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.cov.md (hunk 6 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.cov.md]: 已更换说法

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.cov.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+                  fweights=None,
+                  aweights=None,
+                  name=None)
+```
+
+仅 paddle 参数更多，具体如下：
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| <font color='red'> correction </font>    | <font color='red'> ddof </font>          | 样本量和样本自由度之间的差异， 若为 True ，返回无偏估计结果；若为 False ，返回普通平均值计算结果。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.cov.md:23]: torch支持非bool，这里没有写

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.cov.md [hunk 8]
```diff
@@ -19,12 +19,12 @@ paddle.linalg.cov(x,
...
 
-两者功能一致且参数用法一致，仅参数名不同，具体如下：
-### 参数差异
+仅 paddle 参数更多，具体如下：
+### 参数映射
 | PyTorch       | PaddlePaddle | 备注                                                   |
 | ------------- | ------------ | ------------------------------------------------------ |
 | <font color='red'> input </font>         | <font color='red'> x </font>            | 输入的 Tensor 。                   |
-| <font color='red'> correction </font>    | <font color='red'> ddof </font>          | 若为 True ，返回无偏估计结果；若为 False ，返回普通平均值计算结果。 |
+| <font color='red'> correction </font>    | <font color='red'> ddof </font>          | 样本量和样本自由度之间的差异， 若为 True ，返回无偏估计结果；若为 False ，返回普通平均值计算结果。|
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.cov.md:27]: torch支持非bool，这里没有写

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.norm.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+#### dtype：表示输出 Tensor 的数据类型
+
+```python
+# Pytorch 写法
+torch.linalg.norm(x, dtype=torch.float64)
+
+# Paddle 写法
+paddle.linalg.norm(x).astype(paddle.float64)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.norm.md]: infoflow 2023-06-06 11-36-01

cast 输入x，这种dtype以后都关注下

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.cov.md (hunk 7 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.cov.md:23]: paddle 也支持，支持官网没有说明，实测能对齐

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.cov.md (hunk 8 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.cov.md:27]: 同 torch.Tensor.cov

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.norm.md (hunk 9 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.norm.md]: done!

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.cov.md (hunk 7 above)
Comment (zhwesky2010) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.cov.md:23]: > paddle 也支持，支持官网没有说明，实测能对齐

OK

**Review by zhwesky2010 — APPROVED** [review]
LGTM
